No longer completes classes on Android

## Actual behavior (Required!)
When attempting to complete an import in an Android project, Javacomplete says "packages still empty, try later. indexing..." even on non-android imports

Also related behavior, when completing within a class that extends an Android component there are missing completions when completing ```this``` or ```super```

## Expected behavior (Required!)
Should give imports and complete android related classes

## The steps to reproduce actual behavior (Required!)
 1. Create a basic Android Studio project or use [Plaid](https://github.com/nickbutcher/plaid)
 2. Open the MainActivity.java or other file that extends an Activity
 3. Attempt to complete an android import
 4. ```:messages``` will show that the app is indexing, even after several minutes
 5. Move to class body and attempt completing ```this``` or ```super``` and receive no results or ones that are missing

## Environment (Required!)
 * OS: Archlinux
 * Neovim version: Neovim 1.3.0-68-gd19af41

## Q&A
- [x] Yes, I tried minimal .vimrc configuration.
- [x] Yes, I have enabled logs with `JCdebugEnableLogs` and can put here content of `JCdebugGetLogContent` command, if you need.
- [x] Even, if you wish, I can set `g:JavaComplete_JavaviDebug` to `1`, than set `g:JavaComplete_JavaviLogfileDirectory`, and put here server logs, too.
- For clarification my minimal vim was simply setting runtimepath to the location of vim-javacomplete2 and setting the omnifunc

## Additional information

I'm opening this issue to solicit insight from others and begin tracking down what exactly is causing this. I'm happy to provide additional information, and I suspect that this is not Android specific but I'm only getting these issues within Android projects right now

@DonnieWest 
`./gradlew build` show me 
```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/wsdjeg/forks/plaid/app/build.gradle' line: 17

* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin [id 'com.android.application']
   > Gradle version 2.10 is required. Current version is 2.8. If using the gradle wrapper, try editing the distributionUrl in /home/wsdjeg/forks/plaid/gradle/wrapper/gradle-wrapper.properties to gradle-2.10-all.zip

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.036 secs
```

test with your test project
./gradlew build shows
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugCompile'.
   > Could not find com.android.support:appcompat-v7:23.2.1.
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/support/appcompat-v7/23.2.1/appcompat-v7-23.2.1.pom
         https://jcenter.bintray.com/com/android/support/appcompat-v7/23.2.1/appcompat-v7-23.2.1.jar
     Required by:
         Sample-Project-for-Javacomplete-225:app:unspecified

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 7.942 secs
``` 
but I have installed the support lib in sdk
![2016-03-18 21-02-06](https://cloud.githubusercontent.com/assets/13142418/13878479/ba30c75e-ed4c-11e5-907d-c36288c94cf9.png)

--- a/classpath.gradle
+++ b/classpath.gradle
@@ -8,7 +8,7 @@ task classpath << {
     def listFiles = new File(exploded)
     if (listFiles.exists()) {
       listFiles.eachFileRecurse(){ file ->
-        if (file.name.endsWith("classes.jar")){
+        if (file.name.endsWith(".jar")){
           classpathFiles += file
         }
       }

